### ISSUE TITLE
Simplify backend

### ISSUE THREAD

**Issue by superdyzio** [superdyzio/lunch-voting-token#5]
Most of the server code is no longer needed since all the transactions are signed using Metamask. After #4 it should be possible to remove also part of the backend code responsible for serving webpage. Investigate the code and clean up.

**Expected result:**
Leave only required endpoints in the backend part.
